modules/core/core-api/src/main/java/com/enonic/xp/content/CreateContentParams.java

```diff
@@ -378,9 +381,12 @@ public Builder createSiteTemplateFolder( final boolean createSiteTemplateFolder
             return this;
         }
 
-        public Builder inherited( final boolean inherited )
+        public Builder inherit( final Collection<ContentInheritType> inherit )
         {
-            this.inherited = inherited;
+            if ( inherit != null )
```

Comment:
Why do you need null check?

Reply:
to avoid NPE if someone will send it here(CreateContentParams is PublicApi)